### ISSUE TITLE
The live region service does not allow screen reader users to read messages that were not announced

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4380]
## Describe the bug

[Test URL]
http://cx-guidelines.vmware.com/Clarity3LiveRegionServiceTest/
[WCAG 2.1 Guideline]

[OS]
Windows 10, macOS

[User Agents/Assistive Technologies]
Windows/Firefox/NVDA, macOS/Safari/VoiceOver

[Component]
ARIA Live Service

[Description]
The live region service does not allow screen reader users to read messages that were not announced. 

[User Impact]
When live regions are updated rapidly, some updates may not be announced by the screen reader. Since our current implementation overwrites the entire contents of the live region with every update, screen reader users cannot read the updates with the reading cursor as expected. 

[Steps to Reproduce]
- Go to the Test URL using one of the listed screen reader/browser combinations
- Activate "update live region" button in rapid succession
- Observe that some messages that were displayed visually were not announced by the screen reader
- Observe that you cannot read those messages you didn't hear

[Recommendation]
Provide an implementation that saves the updates so they can be read by screen reader users with reading cursor.

See 
http://cx-guidelines.vmware.com/basicAngularLiveRegion/

Consider letting the consuming application provide the element with aria-live and letting this service update that container, by adding or removing content. This would allow the consuming applications to manage their own live region strategy.
